timemachine/cpp/src/nonbonded_all_pairs.cu

```diff
@@ -270,13 +272,15 @@ void NonbondedAllPairs<RealType, Interpolated>::execute_device(
     // identify which tiles contain interpolated parameters
 
     const int tpb = 32;
-    const int B = (N + tpb - 1) / tpb;
-
-    dim3 dimGrid(B, 3, 1);
```

Comment:
Since there are now 2 different shapes used here (e.g. `dim3(ceil_divide(N, tpb), 3, 1)` vs. `dim3(ceil_divide(K_, tpb), 3,  1`, I opted to just define the shape parameter directly at the call site (I wrote a couple frustrating bugs along the way do to confusing the 2 shapes)

Reply:
It seems like a docstring explaining the difference between K and N would be handy for returning to this code later.

Reply:
> It seems like a docstring explaining the difference between K and N

@badisa I think this is addressed by db146c6 (but let me know if you had something else in mind)

Reply:
Looks good